bt/cmd/rds_gui

```diff
@@ -272,10 +270,6 @@ arg_parse() {
 
 arg_parse
 
-iam_user=restricted  # default
-
-#debug echo instance: ${instance}
-[[ "$help" == true ]] && usage
```

Comment:
iam_user is redefined 5 lines below.  Help var is never set to true

Reply:
Great catch!   Thanks again for this.